tests/ops/test_qubit_ops.py

```diff
@@ -1397,7 +1437,8 @@ def test_single_excitation_minus_generator(self, phi):
         assert np.allclose(res, exp)
 
     @pytest.mark.parametrize(
-        "excitation", [qml.SingleExcitation, qml.SingleExcitationPlus, qml.SingleExcitationMinus]
+        "excitation",
```

Comment:
Most of the changes from here in this file could be reverted to shrink the scope  of the PR.